Customization/T280/PhoneRepairShop_Code/PhoneRepairShop_Code.Tests/PhoneRepairShop_Code.Tests.csproj

```diff
@@ -36,21 +36,24 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="PX.Common">
-      <HintPath>..\..\..\..\..\PhoneRepairShop\Bin\PX.Common.dll</HintPath>
+      <HintPath>..\..\..\..\Bin\PX.Common.dll</HintPath>

```

Comment:
Why paths to dlls have been changed?

Reply:
In fact, they have not changed. The old and new paths point to the same folder.